### DISCUSSION TITLE
Logout after Login

### DISCUSSION THREAD

**Discussion by devPhpMostafa** [luyadev/luya#2180] [Questions & Problems] [answered]
Hello team, i have an issue with luya admin auto logout after login or browsing on any admin module.
with nginx server implementation :

```
  location ~ \.php$ {
    include fastcgi_params;
    fastcgi_intercept_errors on;
    fastcgi_index index.php;
    fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
    try_files $uri =404;
    fastcgi_read_timeout 3600;
    fastcgi_send_timeout 3600;
    fastcgi_param HTTPS "on";
    fastcgi_param SERVER_PORT 443;
    fastcgi_pass 127.0.0.1:{{php_fpm_port}};
    fastcgi_param PHP_VALUE "{{php_settings}}";
    fastcgi_param HTTP_AUTHORIZATION $http_authorization;
    fastcgi_pass_header Authorization; 
  }
```

**Comment by nadar** [comment, marked as answer]
hi @devPhpMostafa 

+ read this section: https://luya.io/guide/installation/problems.html#admin-module-logout-after-login
+ show us your composer.json (versions of luya? latest?)
+ there you will find a nginx php fpm docker image config which works with luya https://gitlab.com/zephir.ch/foss/luya-docker/-/tree/php81/docker/etc

(ps: please format your code accordingly and there is no need to mention me)

**Reply by devPhpMostafa** [reply]
my composer file:
{
    "name": "luyadev/luya-kickstarter",
    "description": "The kickstarter project for a new LUYA application.",
    "type": "luya-kickstarter",
    "license":"MIT",
    "require": {
        "luyadev/luya-module-admin" : "^4.0",
        "luyadev/luya-module-cms" : "^4.0",
        "luyadev/luya-bootstrap3" : "^1.0",
        "luyadev/luya-generic" : "^1.0"
    },
    "require-dev": {
        "yiisoft/yii2-gii" : "^2.0",
        "yiisoft/yii2-debug" : "^2.0",
        "luyadev/luya-testsuite" : "^2.0"
    },
    "scripts": {
        "post-create-project-cmd": [
            "yii\\composer\\Installer::postCreateProject"
        ]
    },
    "extra": {
        "yii\\composer\\Installer::postCreateProject": {
            "setPermission": [
                {
                    "public_html/assets": "0777",
                    "public_html/storage": "0777",
                    "runtime" : "0777"
                }
            ]
        },
        "asset-installer-paths": {
            "bower-asset-library": "vendor/bower"
        },
        "branch-alias": {
            "dev-master": "1.0.x-dev"
        }
    },
    "config": {
        "fxp-asset": {
            "enabled": false
        },
        "allow-plugins": {
            "yiisoft/yii2-composer": true,
            "luyadev/luya-composer": true
        }
    },
    "repositories": [
        {
            "type": "composer",
            "url": "https://asset-packagist.org"
        }
    ]
}

**Reply by nadar** [reply]
please format your code. looks like latest version. so see the problem description link on luya.io

**Reply by devPhpMostafa** [reply]
thanks you . problem is solved after upgrade and set $logoutOnUserIpChange = false instead of true
